src/logic/safe/store/reducer/moduleTransactions.ts

```diff
@@ -0,0 +1,32 @@
+import { handleActions } from 'redux-actions'
```

Comment:
why do we need a separate reducer? what was wrong with the previous one we used for transactions?

Reply:
it's a different endpoint, and the returned data is quite different. Same strategy as with `incomingTransactions`

Reply:
But in the end, we get to merge them to a single list

Reply:
> But in the end, we get to merge them to a single list

yes, but closer to the component.

Fortunately, this will work differently with the `allTransactions` endpoint approach.

In fact, I explored the strategy of storing the plain DTO in the store and then calculate data in the selector, it ended up working quite decently.

Reply:
Then should've gone for it :+1: